app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -50,7 +50,7 @@ echo "#"
 if [ x$ID = xrhel ] || [ x$ID = xcentos ]; then
     register_katello_host(){
         UUID=$(subscription-manager identity | head -1 | awk '{print $3}')
-        curl --cacert $SSL_CA_CERT --request POST "<%= foreman_server_url %>/register?uuid=$UUID" \
+        curl --cacert $SSL_CA_CERT --request POST "<%= @registration_url %>?uuid=$UUID" \
```

Comment:
I see we're still appending `?uuid=$UUID` here, this PR does not make it worse so I'm fine with that. We could play with URI#query, but let's keep that to when we really need it.